src/reducers/costs.tsx

```diff
@@ -4,6 +4,7 @@ import { ICost } from '../types/costs';
 import { updateReductions } from './reductions';
 
 export function applyCost(oldState: IStoreState, cost: ICost): IStoreState {
+  if (typeof cost === 'undefined') { throw 'Internal error - undefined cost'; }
```

Comment:
I don't know why right now, but with the test script, tslint did not warn.
Can be redundant.